lib/browser/app.js

```diff
@@ -59,6 +59,12 @@ app.controller('AppController', function($q, DriveScannerService, SelectionState
     this.selection.clear();
     this.writer.setProgress(0);
     this.scanner.start(2000);
+
+    this.scanner.scan().then(function (res) {
```

Comment:
I don't like calling this local variable `res`. Maybe something like `drives` or `detectedDrives` to better express it's meaning?